README.md

```diff
@@ -149,7 +149,8 @@ The configuration file is a JSON format file, as seen in the following example:
         "logfile_dir" : "logs/",
         "logfile_rotation_size" : "64K",
         "logfile_max_size" : "640K",
-        "debug_logging" : false
+        "debug_logging" : false,
+        "security_enabled" : false 
```

Comment:
wouldn't a name like "peer_validation" be better?

Reply:
Yes. Thanks for the suggestion!

Reply:
OK, did that refactoring and pushed.